astropy/modeling/tests/test_parameters.py

```diff
@@ -723,7 +723,14 @@ def test__create_value_wrapper(self):
         with pytest.raises(TypeError, match=MESSAGE):
             param._create_value_wrapper(np.add, mk.MagicMock())
         # Good ufunc
-        assert param._create_value_wrapper(np.negative, mk.MagicMock()) == np.negative
+        with mk.patch(
+            "astropy.modeling.parameters._wrap_ufunc", autospec=True
+        ) as mkWrap:
+            assert (
+                param._create_value_wrapper(np.negative, mk.MagicMock())
+                == mkWrap.return_value
+            )
+            assert mkWrap.call_args_list == [mk.call(np.negative)]
```

Comment:
Now that `ufunc` is being wrapped to handle internal vs external units it is difficult to easily "see" it. The patch here is to "see" the `ufunc` passed into the wrapper.